### DISCUSSION TITLE
Scrolling on 4 row LCD broken

### DISCUSSION THREAD

**Discussion by Bankst** [VasilKalchev/LiquidMenu#57] [Q&A]
**Describe the bug**
Setting up a 4 row LCD with lines in the same way as the `J_scrolling_menu` example produces odd scrolling behavior

**To Reproduce**
Add 5 or more LiquidLines, numbered as such
LiquidLine(0, 0);
LiquidLine(0, 1);
LiquidLine(0, 2);
LiquidLine(0, 3);
LiquidLine(0, 3);
LiquidLine(0, 3); etc...

**Version of the library**
1.5

**Comment by VasilKalchev** [comment]
Sorry for the delay.

Are you setting the number of _lines_ the display has, like this:
`<your_liquid_screen_object>.set_displayLineCount(4);`

Also, there must be a function attached to each of the _lines_.
